homeassistant/components/min_max/__init__.py

```diff
@@ -1,6 +1,26 @@
 """The min_max component."""
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
 
-DOMAIN = "min_max"
 PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Min/Max from a config entry."""
+    hass.config_entries.async_setup_platforms(entry, (Platform.SENSOR,))
```

Comment:
Why not use `PLATFORMS`?
```suggestion
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```